console-subscriber/src/builder.rs

```diff
@@ -71,6 +75,8 @@ impl Default for Builder {
             recording_path: None,
             filter_env_var: "RUST_LOG".to_string(),
             self_trace: false,
+            #[cfg(feature = "grpc-web")]
+            cors_layer: None,
```

Comment:
As described below, I think this shouldn't be part of `console-subscriber`.

Reply:
Removed.